core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -497,7 +501,6 @@ public GraphHopper init(GraphHopperConfig ghConfig) {
         flagEncodersString = ghConfig.getString("graph.flag_encoders", flagEncodersString);
         encodedValuesString = ghConfig.getString("graph.encoded_values", encodedValuesString);
         dateRangeParserString = ghConfig.getString("datareader.date_range_parser_day", dateRangeParserString);
-        buildEncodingManagerAndOSMParsers(flagEncodersString, encodedValuesString, dateRangeParserString, profilesByName.values());
```

Comment:
We no longer build the encoding manager in `init()`, because when we are loading we want to create it from the stored properties instead.